generate_workspace/src/main/java/com/google/devtools/build/workspace/output/BzlWriter.java

```diff
@@ -32,27 +35,33 @@
       MethodHandles.lookup().lookupClass().getName());
 
   private final String[] argv;
-  private final Path generatedFile;
+  private final File generatedFile;
```

Comment:
Here is a less hand-wavy answer for why we need to use the File.

With the File approach, we create the PrintStream, using `file.getAbsoluteFile()`. It ends up writing to `~/execroot/migration-tooling/_tmp/BzlWriterTest_1/generate_workspace.bzl` 

If I wanted to replicate the behavior of `file.getAbsoluteFile()`we could use `path.toAbsolutePath().toFile()` rather than `path.toFile()` This would ensure we actually write to the jimfs file. 

However, jimfs.JimfsPath throws an UnsupportedOperationException.